AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/PackageManifestBuilder.swift

```diff
@@ -167,4 +173,18 @@ struct PackageManifestBuilder {
 
         return lines.joined(separator: .newline)
     }
+
+    /// Calls the method to include protocol tests in the manifest.
+    ///
+    ///```
+    ///// Uncomment this line to enable protocol tests
+    ///addProtocolTests()
+    ///```
```

Comment:
The formating seems a bit off to me. Maybe 
```suggestion
    /// // Uncomment this line to enable protocol tests
    /// // addProtocolTests()
    ///```
```

Reply:
Looks weird in code, but avoids the leading space when shown in inline docs.